gauge.go

```diff
@@ -103,6 +107,9 @@ func (g *Gauge) Buffer() []Point {
 				p.Bg |= AttrReverse
 			}
 
+			if g.PercentColorHighlighted != Attribute(uint16max) {
```

Comment:
Initial `.PercentColorHighlighted` with `uint16max` as unset flag is compact, but there is a problem:

If one once set `.PercentColorHighlighted` to a value while later he wants to unset it, he has to be aware that he need a value `^uint16(0)` which is not trivial to him.

We only use up to 256 colours values, which means there are 8 digits spare space, so may be we can take advantage of it (I haven't test passing a `Attribute` >= 256 to termbox, not sure what will happen;))